packages/next/lib/verifyTypeScriptSetup.ts

```diff
@@ -80,7 +80,7 @@ export async function verifyTypeScriptSetup(dir: string): Promise<void> {
         chalk.cyan.bold('typescript'),
         'by running',
         chalk.cyan.bold(
-          isYarn ? 'yarn add typescript' : 'npm install typescript',
+          isYarn ? 'yarn add typescript' : 'npm install --save-dev typescript',
```

Comment:
```suggestion
          isYarn ? 'yarn add --dev typescript' : 'npm install --save-dev typescript',
```

We need the one for yarn to be saved in `devDependencies` as well 😅

Reply:
Good catch